apollo-router/src/graphql/mod.rs

```diff
@@ -165,6 +166,19 @@ impl Error {
     }
 }
 
+// GraphQL spec require that both "line" and "column" are positive numbers.
+// However GraphQL Java and GraphQL Kotlin return `{ "line": -1, "column": -1 }`
+// if they can't determine error location inside query.
+// This function removes such locations from suplied value.
+fn skip_invalid_locations(mut value: Value) -> Value {
+    if let Some(array) = value.as_array_mut() {
+        array.retain(|location| {
+            location.get("line") != Some(&json!(-1)) || location.get("column") != Some(&json!(-1))
```

Comment:
We need to check but I think it would be better to use enum Value directly instead of using the json macro which could parse

Reply:
Fixed ✅